dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/internal/utils/OAuthClientHttpUtil.java

```diff
@@ -308,46 +308,50 @@ void debugPostToEndPoint(String url, @Sensitive List<NameValuePair> params, Stri
         Tr.debug(tc, "CURL Command: " + sb.toString());
     }
 
-    public HttpClient createHTTPClient(SSLSocketFactory sslSocketFactory, String url, boolean isHostnameVerification) {
+    public HttpClient createHTTPClient(SSLSocketFactory sslSocketFactory, String url, boolean isHostnameVerification, boolean useJvmProps) {
 
         HttpClient client = null;
 
         if (url != null && url.startsWith("http:")) {
-            client = HttpClientBuilder.create().build();
+            client = getBuilder(useJvmProps).build();
         } else {
             SSLConnectionSocketFactory connectionFactory = null;
             if (!isHostnameVerification) {
                 connectionFactory = new SSLConnectionSocketFactory(sslSocketFactory, new AllowAllHostnameVerifier());
             } else {
                 connectionFactory = new SSLConnectionSocketFactory(sslSocketFactory, new StrictHostnameVerifier());
             }
-            client = HttpClientBuilder.create().setSSLSocketFactory(connectionFactory).build();
+            client = getBuilder(useJvmProps).setSSLSocketFactory(connectionFactory).build();
         }
 
         return client;
     }
 
-    public HttpClient createHTTPClient(SSLSocketFactory sslSocketFactory, String url, boolean isHostnameVerification, String baUser, @Sensitive String baPassword) {
+    public HttpClient createHTTPClient(SSLSocketFactory sslSocketFactory, String url, boolean isHostnameVerification, String baUser, @Sensitive String baPassword, boolean useJvmProps) {
 
         HttpClient client = null;
 
         BasicCredentialsProvider credentialsProvider = new BasicCredentialsProvider();
         credentialsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials(baUser, baPassword));
 
         if (url != null && url.startsWith("http:")) {
-            client = HttpClientBuilder.create().setDefaultCredentialsProvider(credentialsProvider).build();
+            client = getBuilder(useJvmProps).setDefaultCredentialsProvider(credentialsProvider).build();
         } else {
             SSLConnectionSocketFactory connectionFactory = null;
             if (!isHostnameVerification) {
                 connectionFactory = new SSLConnectionSocketFactory(sslSocketFactory, new AllowAllHostnameVerifier());
             } else {
                 connectionFactory = new SSLConnectionSocketFactory(sslSocketFactory, new StrictHostnameVerifier());
             }
-            client = HttpClientBuilder.create().setDefaultCredentialsProvider(credentialsProvider).setSSLSocketFactory(connectionFactory).build();
+            client = getBuilder(useJvmProps).setDefaultCredentialsProvider(credentialsProvider).setSSLSocketFactory(connectionFactory).build();
         }
 
         return client;
     }
+    
+    private HttpClientBuilder getBuilder(boolean useJvmProps){
+         return useJvmProps ? HttpClientBuilder.create().useSystemProperties() : HttpClientBuilder.create();
+    }
```

Comment:
Considering this identical method is now in three places, we can probably come back and consolidate the methods into one somewhere in a common security infrastructure project.